### ISSUE TITLE
Stopped working, crashes with python error

### ISSUE THREAD

**Issue by Erudition** [ovenwerks/studio-controls#56]
I didn't change anything in particular about my setup, but something must have changed because now whenever I try to launch studio controls it never opens. I tested in a terminal to see what's going on:

```python
$ studio-controls
install path: /usr/local
Traceback (most recent call last):
  File "/usr/local/bin/studio-controls", line 1840, in <module>
    us = StudioControls()
  File "/usr/local/bin/studio-controls", line 350, in __init__
    if new_pid != old_pid:
UnboundLocalError: local variable 'old_pid' referenced before assignment
```
Unfortunately, this doesn't seem to go away even with a reinstall, a --purge reinstall, nor using a different version. I even tried using the latest beta, built from source.

It must be based on something on my system, since it persists with all versions, but it's not in the config that gets purged with uninstall at least. Not sure what else to purge. python3 seems to be the latest version. 

Thank you in advance, I really love your tool.

**Comment by ovenwerks** [comment]
I am not sure which version you are using, but the latest beta (2.1.68) fixes this I think.

**Comment by Erudition** [comment]
Unfortunately it did not change anything :/

**Comment by ovenwerks** [comment]
Please test 2.1.70.

**Comment by ovenwerks** [comment]
This should be fixed in 2.2.0

**Comment by Erudition** [comment]
Yes! Thank you!
